modules/custom_operations/README.md

```diff
@@ -0,0 +1,50 @@
+# Collection of Custom Operations using OpenVINO Extensibility Mechanism
+
+This module provides a guide and the implementation a few custom operations in the Intel OpenVINO runtime using its [Extensibility Mechanism](https://docs.openvino.ai/latest/openvino_docs_Extensibility_UG_Intro.html).
+
+More specifically, here we implement custom OpenVINO operations that add support for the following native PyTorch operations:
+
+* [torch.fft](examples/fft)
+* [nn.functional.grid_sample](/examples/grid_sample)
+
+And other custom operations introduced by third-party frameworks:
+
+* [calculate_grid](/examples/calculate_grid) and [sparse_conv](/examples/sparse_conv) from [Open3D](https://github.com/isl-org/Open3D)
+* [complex_mul](/examples/complex_mul) from [DIRECT](https://github.com/NKI-AI/direct)
+
+You can find more information about how to create and use OpenVINO Extensions to facilitate mapping of custom operations from framework model representation to OpenVINO representation [here](https://docs.openvino.ai/latest/openvino_docs_Extensibility_UG_Frontend_Extensions.html).
+
+
+## Build custom OpenVINO operation extension library
+
+The C++ code implementing the custom operation is in the `user_ie_extensions` directory. You'll have to build an "extension library" from this code so that it can be loaded at runtime. The steps below describe the build process:
+
+```bash
+source /opt/intel/openvino_2022/setupvars.sh
+
+cd user_ie_extensions
+mkdir build && cd build
+cmake .. -DCMAKE_BUILD_TYPE=Release && make -j$(nproc --all)
```

Comment:
1. nproc does not work for macOS AFAIK
2. make is not available out of box on Windows. 

Can we just use `cmake --build .. --parallel 4` ?

Reply:
I agree, if `cmake --build` makes this easier across multiple OS platforms, let's use that. Just FYI `nproc` is available on macOS.

Reply:
> Just FYI nproc is available on macOS.

It's not available out of box, only one you've installed `coreutils` from brew. Native way is to use `sysctl -n hw.ncpu`

Reply:
Ah! I did not know that, good to know! Thanks!